packages/html-manager/package.json

```diff
@@ -35,6 +35,7 @@
     "test:unit:firefox:headless": "npm run test:unit:default -- --browsers=FirefoxHeadless"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-free": "^5.12.0",
```

Comment:
FYI, this is not a typo. The org actually is fortawesome: https://github.com/FortAwesome/Font-Awesome/blob/d8db3d032d22e7a5e21f3b40905b7867be19bb02/js-packages/%40fortawesome/fontawesome-free/package.json#L51